tests/plugins/font/manual/fontcommand.md

```diff
@@ -0,0 +1,30 @@
+@bender-tags: 4.14.0, feature, 3306, 3728
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, font, enterkey, elementspath, notification
+
+General Note:
+* Font and font size now use commands instead of direct operation on editor's content.
+* When font or fontSize command is executed then notification should be displayed in the editor and browser's console.
+* Please repeat steps for both font and font size features
+
+# Scenario 1:
+1. Select unstyled text and apply new font/fontSize.
+2. Try to apply this same font/fontSize again.
+3. Try to apply different font for already styled text.
+
+## Expected:
+Command should be executed only once. Styling the same content with the same style should not trigger command.
+
+## Unexpected:
+Command is not executed or is executed more than once.
+
+# Scenario 2:
+1. Select styled text and remove font/fontSize from it.
+2. Try to remove style from not styled text.
```

Comment:
I know it's easier to keep multiple related case studies in the same file to remove HTML/JS code duplication, however, more complicated reproduction steps make harder manual testing, especially on smaller devices. Maybe both scenarios could be extracted into separate files, but with shared `__template__.html` file? It would probably require structure like:

* `manual/command`
  * `__template__.html`
  * `applystyle.md`
  * `removestyle.md`

WDYT?